.github/workflows/cocoapods.yml

```diff
@@ -1,14 +1,17 @@
 name: Check & Update Cocoapods
 
-on:
-  pull_request_target: ~
+on: pull_request_target
+
+permissions:
+  contents: write
+  pull-requests: write
 
 jobs:
   ios-podfile-update:
     # Adapted from https://gist.github.com/A-Tokyo/0d811e818513fc4d3272335d2847d748
     name: iOS Update Cocoapods
     runs-on: macos-11
-    if: startsWith(github.ref_name, 'dependabot/npm_and_yarn/') && contains(github.ref_name, 'react-native') && github.actor == 'dependabot[bot]'
+    if: ${{ startsWith(github.ref_name, 'dependabot/npm_and_yarn/') && contains(github.ref_name, 'react-native') && github.actor == 'dependabot[bot]' }}
```

Comment:
Does this need ${{?

Reply:
I was following GH's documentation. Do you want this removed?

Reply:
Sounds good to me.